src/dmd/builtin.d

```diff
@@ -266,15 +390,15 @@ public extern (C++) void builtin_init()
     add_builtin("_D3std4math4sqrtFNaNbNiNfeZe", &eval_sqrt);
     add_builtin("_D3std4math4fabsFNaNbNiNfeZe", &eval_fabs);
     add_builtin("_D3std4math5expm1FNaNbNiNfeZe", &eval_unimp);
-    add_builtin("_D3std4math4exp21FNaNbNiNfeZe", &eval_unimp);
+    add_builtin("_D3std4math4exp2FNaNbNiNfeZe", &eval_unimp);
```

Comment:
ditto

Reply:
ditto tree

Reply:
Likewise, not ditto. :-)

Reply:
std.math has annotated exp2 with `@trusted`, so this overload can go in favor of the one a few lines below.